integration/v7/isolated/app_command_test.go

```diff
@@ -272,20 +272,6 @@ applications:
 							Eventually(session1).Should(Exit(0))
 						})
 					})
-					When("the deployment is cancelled", func() {
```

Comment:
Any particular reason to remove this test? Are we testing this functionality in a different place?

Reply:
There's a small window when `cf app` will report back a rolling deployment is canceling, and even between successive calls this can be missed which leads to a flakey test. I think this is covered by [unit tests for app_summary_displayer](https://github.com/cloudfoundry/cli/blob/main/command/v7/shared/app_summary_displayer_test.go#L825-L874).